v3/src/components/case-table/checkbox-cell.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef } from "react"
+import { useDataSetContext } from "../../hooks/use-data-set-context"
+import { FValue } from "../../models/formula/formula-types"
+
+export const isBoolean = (value: FValue | undefined) => {
+  let v = value
+  if (typeof v === "string") {
+    v = v.toLowerCase()
+  }
+  return v === undefined || ["", "false", "true", false, true, null].includes(v as string | boolean | null)
+}
+
+interface ICheckboxCellProps {
+  rowId: string
+  attrId: string
+}
+
+export default function CheckboxCell ({ rowId, attrId }: ICheckboxCellProps) {
```

Comment:
```suggestion
export function CheckboxCell ({ rowId, attrId }: ICheckboxCellProps) {
```
We've been avoiding default imports lately in favor of named imports.